subsystems/SmartDifferentialDriveTrain.java

```diff
@@ -0,0 +1,88 @@
+package frc.robot.core751.subsystems;
+
+import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+
+import edu.wpi.first.wpilibj.PWMVictorSPX;
+import edu.wpi.first.wpilibj.SpeedController;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.core751.wrappers.WCANSparkMax;
+
+public class SmartDifferentialDriveTrain extends SubsystemBase {
```

Comment:
would it be possible for you to make this extend DifferentialDriveTrain to make the code base a little nicer/less duplicated?